src/modules/lesson/controllers/lesson-hashtag.controller.ts

```diff
@@ -0,0 +1,195 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpStatus,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import {
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+  PickType,
+} from '@nestjs/swagger';
+import { ModelName } from '@src/constants/enum';
+import { ApiFailureResponse } from '@src/decorators/api-failure-response.decorator';
+import { ApiSuccessResponse } from '@src/decorators/api-success-response.decorator';
+import { BearerAuth } from '@src/decorators/bearer-auth.decorator';
+import { SetModelNameToParam } from '@src/decorators/set-model-name-to-param.decorator';
+import { UserLogin } from '@src/decorators/user-login.decorator';
+import { IdRequestParamDto } from '@src/dtos/id-request-param.dto';
+import { JwtAuthGuard } from '@src/guards/jwt-auth.guard';
+import { OptionalJwtAuthGuard } from '@src/guards/optional-auth-guard';
+import { PrismaHelper } from '@src/modules/core/database/prisma/prisma.helper';
+import { CreateManyHashtagDto } from '@src/modules/hashtag/dtos/create-many-hashtag.dto';
+import { LessonHashtagParamDto } from '@src/modules/hashtag/dtos/hashtag-param.dto';
+import { UpdateHashtagDto } from '@src/modules/hashtag/dtos/update-hashtag.dto';
+import { UpdateManyHashtagDto } from '@src/modules/hashtag/dtos/update-many-hashtag.dto';
+import { LessonHashtagEntity } from '../entities/lesson-hashtag.entity';
+import { LessonEntity } from '../entities/lesson.entity';
+import { LessonHashtagService } from '../services/lesson-hashtag.service';
+
+@ApiTags('과제의 해시태그')
+@Controller(':id/hashtags')
+export class LessonHashtagController {
+  constructor(
+    private readonly lessonHashtagService: LessonHashtagService,
+    private readonly prismaHelper: PrismaHelper,
+  ) {}
+
+  @ApiOperation({ summary: '과제 해시태그 생성' })
+  @ApiOkResponse({ type: PickType(LessonEntity, ['hashtags']) })
+  @ApiFailureResponse(HttpStatus.FORBIDDEN, 'You do not have access to ~')
+  @ApiFailureResponse(HttpStatus.NOT_FOUND, "~ doesn't exist id in ~")
```

Comment:
~ 로 표현하는거보다는 어떤 변수명으로 표현해주는게 좋겠어요
**'You do not have access to {modelName}'**
저기에 들어가는 변수명은 클라이언트 측에서 알아볼 수 있게끔 변수명을 짓는게 좋지 않을까 싶네요.

Reply:
아래 더 보니 모든 path 에 대해 이 두개가 들어가는데  class 에 붙이거나 아니면 계속 복붙을 하고 수정을 하게되면 일일이 수정해야하니 하나의 클레스로 만들어서 사용하는것도 방법이겠어요.

Reply:
좋은 조언 감사드립니다.
constant.ts 파일에 HTTP에러 메시지를 메핑해주는 HTTP_ERROR_MESSAGE를 추가한 뒤 
```
export const HTTP_ERROR_MESSAGE = {
  FORBIDDEN: 'You do not have access to {modelName}',
  NOT_FOUND: "{modelId} doesn't exist id in {modelName}",
};
```

```suggestion
   @ApiFailureResponse(HttpStatus.FORBIDDEN, HTTP_ERROR_MESSAGE.FORBIDDEN)
  @ApiFailureResponse(HttpStatus.NOT_FOUND, HTTP_ERROR_MESSAGE.NOT_FOUND)
```
이런식으로 사용될 수 있도록 수정하겠습니다!